pygeos/io.py

```diff
@@ -7,26 +8,48 @@
 from . import geos_capi_version_string
 
 
-shapely_geos_version = None
 ShapelyGeometry = None
+shapely_compatible = None
+shapely_lgeos = None
+shapely_geom_factory = None
+shapely_wkb_loads = None
 _shapely_checked = False
 
 def check_shapely_version():
-    global shapely_geos_version
     global ShapelyGeometry
+    global shapely_compatible
```

Comment:
Can you document here that `shapely_compatible` can be `None`, `True` and `False`?

Reply:
I added some documentation to the `check_shapely_version()` function